src/grt/src/fastroute/src/utility.cpp

```diff
@@ -2495,8 +2495,8 @@ void FastRouteCore::saveCongestion(const int iter)
     const int seed = 42;
     g.seed(seed);
 
-    utl::shuffle(congestionGridsV.begin(), congestionGridsV.end(), g);
     utl::shuffle(congestionGridsH.begin(), congestionGridsH.end(), g);
+    utl::shuffle(congestionGridsV.begin(), congestionGridsV.end(), g);
```

Comment:
Is this just a leftover or is there some significance to the order of these lines?

Reply:
Just a leftover.